main.js

```diff
@@ -1,10 +1,9 @@
-import { app, BrowserWindow, ipcMain, Menu, Tray } from 'electron';
+import { electron, app, BrowserWindow, ipcMain, Menu, Tray } from 'electron';
```

Comment:
if this is what you had before:

```
const electron = require('electron');
```

I'm pretty sure you want this here, since electron is the default export of that module:

```
import electron, { app, BrowserWindow, ipcMain, Menu, Tray } from 'electron';
```

But, if the way you have works (it doesn't bomb when you reference electron in the code), then it's ok.